docs/developer-guide/ksqldb-reference/create-stream-as-select.md

```diff
@@ -17,7 +17,10 @@ CREATE [OR REPLACE] STREAM stream_name
   [WITH ( property_name = expression [, ...] )]
   AS SELECT  select_expr [, ...]
   FROM from_stream
-  [[ LEFT | FULL | INNER ] JOIN [join_table | join_stream] [ WITHIN [(before TIMEUNIT, after TIMEUNIT) | N TIMEUNIT] ] ON join_criteria]* 
+  [[ LEFT | FULL | INNER ]
+      JOIN [join_table | join_stream]
+          [WITHIN [<size> <timeunit> | (<before_size> <timeunit>, <after_size> <timeunit>)] [, GRACE PERIOD <grace_size> <timeunit>]]
```

Comment:
Is the `,` correct?

I though we do `WITHIN 5 SEC GRACE PERIOD 3 SEC` without `,` before `GRACE` ?

Reply:
Aaah, true. I left it accidentally. I'll remove them .